internal/olm/operator/internal/configmap.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
```

Comment:
Since you split the impl and create one go file for each kind of secondary resource which is required think that them should be inside of a new dir in the path for we easy know that these files are related to the feature X. Is it make sense?

Reply:
It could be `internal/olm/registry`, but this package is only intended to be used by [code](https://github.com/operator-framework/operator-sdk/pull/1912/files#diff-59b535800ca0d926da38b8e79bc87818R267) in `internal/olm/operator`, hence its current path. If that changes in the future we can move it. I recommend going through #1912 to get a handle on why this code is set up as-is.

Reply:
OK. But it shows strange internal/.../internal again.
WDYT about `internal/olm/operator/registry`

Reply:
It isn't a common use case but it is a completely viable use of `internal`.